src/NetTopologySuite/NtsGeometryServices.cs

```diff
@@ -69,19 +74,8 @@ public static NtsGeometryServices Instance
         /// <summary>
         /// Gets or sets a value indicating the operations to use for geometry overlay.  
         /// </summary>
-        public GeometryOverlay GeometryOverlay
-        {
-            get { return _geometryOverlay ?? (_geometryOverlay = DefaultGeometryOverlay); }
-            set
-            {
-                if (_geometryOverlay != null)
-                    throw new InvalidOperationException();
-                if (value == null)
-                    value = DefaultGeometryOverlay;
-
-                _geometryOverlay = value;
-            }
-        }
+        /// <returns>A set of geometry overlay functions.</returns>
+        internal GeometryOverlay GeometryOverlay { get; }
```

Comment:
IMO this was fine as `public` in this class, since this class is pretty much just a bag of properties anyway.